frontend-react/src/components/annotate.js

```diff
@@ -41,17 +42,30 @@ class Annotate extends Component {
         }
 
         return (
-            <div className="backgroundAll">
+            <div className="border">
                 <form className="selectForm" onSubmit={this._handleSubmit}>
                     <input type="file" id="file" className="fileInputButton" onChange={this._handleImageChange} />
                     <label for="file" class="fileLabel">Select Image</label>
-                    {/* <button type="submit" onClick={this._handleSubmit}>Upload Image</button> */}
+                    {/* <button type="submit" onClick={this._handleSubmit}>Send Annotations</button> */}
                 </form>
                 <div className="imageDisplay">
+
+                    {/* Currently has hardcoded images, needs to take in uploaded images */}
+                    
+                    {/* <ReactImageAnnotate
+                        selectedImage="https://i.imgur.com/SY4z4Cx.jpg"
+                        taskDescription="# Draw region around each Kiwifruit bunch"
+                        images={[{ src: "https://i.imgur.com/SY4z4Cx.jpg", name: "Kiwifruit.png" }]}
+                        regionClsList={["Kiwifruit", "Human Face"]}
+                        enabledTools="create-polygon, create-box"
+                        // onExit = {console.log(ReactImageAnnotate.MainLayoutState)}
+                    /> */}
+
```

Comment:
Dead code shouldn't really be in the commit.

Reply:
This isn't dead code, it is to show the functionality of the library which is necessary for future additions to the code, such as what Andrei is working on in parallel to my work.